src/openms/source/ANALYSIS/ID/AccurateMassSearchEngine.cpp

```diff
@@ -270,6 +270,7 @@ namespace OpenMS
   found_adduct_(),
   empirical_formula_(),
   matching_hmdb_ids_(),
+  mass_trace_intensities_(),
   isotopes_sim_score_(-1.0)
```

Comment:
from what I can see, the order is ok (equal to decl in header).
The build failed because int is compared to Size.